--- configure.ac	2017-10-01 13:51:12.977629338 +0300
+++ configure.ac	2017-10-01 13:55:07.341454777 +0300
@@ -278,7 +278,6 @@
 
 AC_CONFIG_HEADERS([reldap_autoconf.h])
 AC_CONFIG_HEADERS([include/ldap_features.h])
-AC_CONFIG_HEADERS([include/ldap_dirs.h])
 AC_CONFIG_HEADERS([include/lber_types.h])
 
 dnl ================================================================
--- include/Makefile.am	2017-10-01 13:50:03.358372558 +0300
+++ include/Makefile.am	2017-10-01 13:51:39.000000000 +0300
@@ -30,8 +30,9 @@
 BUILT_SOURCES = ldap_dirs.h
 CLEANFILES = ldap_dirs.h
 EXTRA_DIST = ldap_dirs.h.in
+LDAP_DIRS = $(srcdir)/ldap_dirs.h.in
 
-ldap_dirs.h: $(srcdir)/ldap_dirs.h.in Makefile
+ldap_dirs.h: $(LDAP_DIRS) Makefile
 	@if $(AM_V_P); then set -x; else echo "  GEN      $@"; fi; \
 	$(RM) $@ && \
 	echo "/* Generated from $< on `date --rfc-3339=seconds --utc` */" > $@ && \
